pfSense-pkg-API/files/etc/inc/apicalls.inc

```diff
@@ -19,6 +19,39 @@ api_runtime_allowed();    // Check that our configuration allows this API call t
 session_start();    // Start our session. This is only used for tracking user name
 $pf_ver_num = get_pfsense_version()["program"];    // Pull our full pfSense version
 
+
+function api_system_tables() {
+    # VARIABLES
+    global $err_lib, $config, $api_resp, $client_params;
+    $read_only_action = true;    // Set whether this action requires read only access
+    $req_privs = array("page-all");    // Array of privs allowed
+    $http_method = $_SERVER['REQUEST_METHOD'];    // Save our HTTP method
+
+    # RUN TIME
+    // Check that client is authenticated and authorized
+    if (api_authorized($req_privs, $read_only_action)) {
+        // Check that our HTTP method is GET (READ)
+        if ($http_method === 'GET') {
+            $table_status = [];
+            $table_status["tables"] = get_table_status();
+
+            if (isset($client_params['search'])) {
+                $search = $client_params['search'];
+                $carp_status = api_extended_search($carp_status, $search);
```

Comment:
Minor issue but this variable should probably be renamed to something more applicable to the tables.

Reply:
I used another section the api_status_carp as a template, and did not catch those variables. will modify, good catch.